cmd/alerts/add_silence.go

```diff
@@ -0,0 +1,49 @@
+package alerts
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+)
+
+func NewCmdAddSilence() *cobra.Command {
+	return &cobra.Command{
```

Comment:
Can we add `DisableAutoGenTag: true,` to this command for consistency with the others?